src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -132,6 +132,7 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
       !this.isFirm && !this.isBenBcCccUlc &&
       !this.isCoop)
     conditions[6] = () => (EnumUtilities.isTypeRegistration(this.filing) && !this.isFirm)
+    conditions[7] = () => (EnumUtilities.isTypeAmalgamation(this.filing) && !this.isBenBcCccUlc)
```

Comment:
This is a safety check to ensure that the correction action for this ledger item is disabled if this business is not a supported entity type.